### DISCUSSION TITLE
Gazebo 사용하려고 따라하다보니 파일이 없는데 따로 설치해야할지해서요

### DISCUSSION THREAD

**Discussion by makerPIF** [Road-Balance/gcamp_ros_basic#3] [Q&A]
제가 김수영님 영상보면서 가제보를 해보려고 하다보니.. 
gazebo/gui.ini 파일이 없어서 그러는데 이게 없는건 전체 패키지 설치후에도 업다는건 가제보를 새로 설치해야하는건가요?

**Comment by kimsooyoung** [comment]
그런 아닙니다!! 설치 이후 아무런 모델 추가가 없어서 그럴 것입니다.
해당 파일을 만드시고 하는 방법과, gazebo에서 제공하는 모델을 입력하면 자동 생성되는 방식으로 해결이 가능할 것 같은데요,

저도 검증 후에 이 Discussion과 강의 노트 모두에 추가토록 하겠습니다.
부족한 강의에 피드백 남겨 주셔서 감사드리며 빠른 답변 달도록 하겠습니다. 👍👍

**Reply by kimsooyoung** [reply]
강의 노트 업데이트 되었음
